modules/runner/src/testFixtures/java/org/apache/ignite/internal/Cluster.java

```diff
@@ -348,9 +352,9 @@ public Ignite startNode(int index, String nodeBootstrapConfigTemplate) {
         Ignite newIgniteNode;
 
         try {
-            IgniteServer node = startEmbeddedNode(index, nodeBootstrapConfigTemplate);
-            node.waitForInitAsync().get(20, TimeUnit.SECONDS);
-            newIgniteNode = node.api();
+            ServerRegistration registration = startEmbeddedNode(index, nodeBootstrapConfigTemplate);
+            registration.registrationFuture().get(20, TimeUnit.SECONDS);
```

Comment:
Why not "willSucceedIn"? Then we could get rid of exceptions 

```suggestion
        assertThat(registration.registrationFuture(), willSucceedIn(20, TimeUnit.SECONDS));
```